trlx/utils/__init__.py

```diff
@@ -68,42 +70,46 @@ class OptimizerName(str, Enum):
     ADAMW: str = "adamw"
     ADAM_8BIT_BNB: str = "adam_8bit_bnb"
     ADAMW_8BIT_BNB: str = "adamw_8bit_bnb"
+    RMSPROP: str = "rmsprop"
     SGD: str = "sgd"
 
 
-def get_optimizer_class(name: OptimizerName):
+torch_optimizers: Dict[str, type] = dict(
```

Comment:
@jon-tow I remember you being somewhat against this?

Reply:
This should be fine. We don't really care about much of the torch optimizers outside of the current ones. I'm not even sure we should supply RMSProp as it's never really used for optimizing transformer language models.

Reply:
This should be fine. We don't really care about much of the torch optimizers outside of the current ones. I'm not even sure we should supply RMSProp as it's never really used for optimizing transformer language models.